solarforecastarbiter/io/tests/test_api.py

```diff
@@ -1256,6 +1261,7 @@ def auth_token():
                   '"password": "Thepassword123!", '
                   '"audience": "https://api.solarforecastarbiter.org", '
                   '"client_id": "c16EJo48lbTCQEhqSztGGlmxxxmZ4zX7"}'))
+        token_req.raise_for_status()
```

Comment:
@alorenzo175 more scope creep but this fixes the tests when the api/auth0 isn't available